editor/package.json

```diff
@@ -171,6 +172,7 @@
     "object-path-immutable": "3.0.0",
     "onigasm": "2.2.4",
     "path": "0.12.7",
+    "path-browserify": "^1.0.1",
```

Comment:
Can we remove the `^` from the new dependencies please

Reply:
Fixed